frontend/packages/client/src/components/Community/CommunityEditorLinks.js

```diff
@@ -37,149 +59,37 @@ export const CommunityLinksForm = ({
           </div>
         </div>
       </div>
-      <div
-        style={{ position: "relative" }}
-        className="is-flex is-align-items-center mt-4"
-      >
-        <input
-          type="text"
-          name="web"
-          className="rounded-sm border-light py-3 pr-3 column is-full"
-          value={fields["websiteUrl"]}
-          maxLength={200}
-          onChange={(event) =>
-            onChangeHandler["websiteUrl"](event.target.value)
-          }
-          style={{
-            paddingLeft: "34px",
-          }}
-          disabled={isUpdating}
-        />
+      {FormFields.map((formField) => (
         <div
-          className="pl-3"
-          style={{
-            position: "absolute",
-            height: 18,
-            opacity: 0.3,
-          }}
+          style={{ position: "relative" }}
+          className="is-flex is-align-items-center mt-4"
```

Comment:
this should include a `key` prop to resolve the react warning

Reply:
fixed, also removed the dashed line when image is added